renderer/stores/root.ts

```diff
@@ -0,0 +1,13 @@
+import { ConsoleStore } from './console';
+// Make sure store imports are in alphabetical order
+
+class RootStore_ {
+  console: ConsoleStore;
+
+  constructor() {
+    this.console = new ConsoleStore(this);
+    /** Initialize more stores here (try to keep it in alphabetical order) */
```

Comment:
After defining a Store, make sure to create an instance of it in the `RootStore` here.